### PR TITLE
LIVE-3724 Set Ethereum minimum app version to v1.9.20

### PR DESCRIPTION
**Pull request by haammar-ledger**
### 📝 Description

Two separate features break retro-compatibility with current Ethereum Nano app version:
- Support of [EIP-712](https://ledgerhq.atlassian.net/browse/LIVE-3139)
- Support of [EVM light integration](https://ledgerhq.atlassian.net/browse/LIVE-1012)

Both require minimum Ethereum Nano app v1.9.20

:warning: **Note: need to make sure this app version will be available on provider 1 before LL is release with this PR.**

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-3724 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by henrily-ledger — APPROVED** [review]

File: libs/ledger-live-common/src/apps/support.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ const appVersionsRequired = {
   Algorand: ">= 1.2.9",
   Polkadot: ">= 13.9250.0",
   Elrond: ">= 1.0.11",
-  Ethereum: ">= 1.9.17",
+  Ethereum: ">= 1.9.20",
```
Comment (gre) [inline libs/ledger-live-common/src/apps/support.ts:36]: do you confirm this version is available on latest firmware of all devices? (nano s, nano x, ..)

if so, it's good.

**Review by gre — APPROVED** [review]

File: libs/ledger-live-common/src/apps/support.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledger-live-common/src/apps/support.ts:36]: We will merge when the version is ready, for now it's now available I guess

**Comment by gre** [comment]
Please also run the bot against this PR.
when we change the minimum app version we must ALWAYS run the bot, because we saw for instance that we recently merged Zcash which broke the bot.
bot uses the min app version to correctly filter the app and we may have bad surprises!

**Comment by lambertkevin** [comment]
> Please also run the bot against this PR. when we change the minimum app version we must ALWAYS run the bot, because we saw for instance that we recently merged Zcash which broke the bot. bot uses the min app version to correctly filter the app and we may have bad surprises!

Unfortunately we can't as the app isn't released yet and we have to merge it in order to have it in the next release... :(

**Comment by gre** [comment]
we must not optimistically do changes in Ledger Live. instead we must wait the version is ready, run it against the bot, and make sure it pass the bot (using that version)

i would advise to revert the change, because isn't this going to block the usability of the app?

**Pull request merged**